akka-stream/src/main/scala/akka/stream/javadsl/JsonFraming.scala

```diff
@@ -35,6 +35,6 @@ object JsonFraming {
    *                            this Flow will fail the stream.
    */
   def bracketCounting(maximumObjectLength: Int): Flow[ByteString, ByteString, NotUsed] =
-    akka.stream.scaladsl.JsonFraming.bracketCounting(maximumObjectLength).asJava
+    akka.stream.scaladsl.JsonFraming.objectScanner(maximumObjectLength).asJava
```

Comment:
I googled some more and thought that it really is a scanner (like first phases of a lexer)